common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/CommandParameters.java

```diff
@@ -60,9 +60,8 @@
     @Expose()
     private String requiredBrokerProtocolVersion;
 
-    @Builder.Default
     @Expose()
-    private SdkType sdkType = SdkType.MSAL;
```

Comment:
Do you need to get rid of this parameter?

Reply:
yes because this was preventing picking up the value when I checked using the delombok'd code.

Reply:
Do we know why it wasn't being picked up? because you said earlier that when `null` was passed then the default was working as expected and setting it to `MSAL` -- so in what cases does it not work? and why?

Reply:
No, I meant that if the field is not sent over the wire, we have nothing to compare against null on the receiving end. The annotation as well wasn't a good addition, it was a part of an experimentation which I was trying out, and missed out on actually servicing it.

Reply:
But if there is nothing, isn't there the initial value of `null` for all java objects? I mean the default should be kicking in whether you pass `null` to the builder or pass nothing

Reply:
Is this coming through Parcelable?

Reply:
The Builder.Default is broken according to sum, and seems like people are recommending against using it. This might explain the builder not picking up the value passed.
https://github.com/rzwitserloot/lombok/issues/1347

Reply:
@AdamBJohnsonx  I dont think its coming through parcelable.